### PR TITLE
Portfolio finish mobile version

### PR DESCRIPTION
**Pull request by tGemechis7**
I have created the work section , About myself and other in this project using HTML, CSS, JavaScript
i have created html part with flexbox, Grid css
this project makes me best software developer as ajunior

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,34 +1,211 @@
...
+            <li class="social-icon github">
+                <a><img src="images/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="images/fingers.png" alt="fingers"></a>
+            </li>
+
+        </ul>
+    </section>
+    <section id="portfolio" class="project-section">
```
Comment (vickymarz) [inline index.html:68]: - [ ] Good job adding the project cards. Kindly note that the project requirement expects that the main container of the projects has a `display:grid`. Therefore use `grid` to align the entire project section. The goal is to get you familiar with `grid` and `flexbox`

Comment (vickymarz) [inline index.html:68]: - [ ] Your project section seems to have a horizontal scrollbar which is not a good user experience and it also affects the responsiveness of your webpage. Kindly fix it 👍🏽 

image

File: main.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+workSections.innerHTML = '';
+for (let i = 0; i < projectsList.length; i += 1) {
+  let languages = '';
+  projectsList[i].technologies.forEach((language) => {
+    languages += `
+      <li><a href='' class='skill' title='${language}' >${language}</a></li>
+    `;
+  });
+  workSections.innerHTML += `
+      <div class='projects'>
```
Comment (vickymarz) [inline main.js:63]: - [ ] Good job aligning your project card with flexbox. Kindly fix the alignment and spacing of the elements of your project cards to match the figma design 👍🏽 

your design 👇🏽 
image
  
figma design 👇🏽 
image

File: index.html [hunk 3]
```diff
@@ -1,34 +1,211 @@
...
+
+    </section>
+    <section id="about" class="about">
```
Comment (vickymarz) [inline index.html:71]: - [ ] Kindly ensure that you align the elements of your about section to match the figma design. Make sure you use the right font-weight. font-size and color of the texts of the about section to match the figma design 💯 , 

Your design 👇🏽 
image

Figma design 👇🏽 
image

File: index.html [hunk 4]
```diff
@@ -1,34 +1,211 @@
...
+                <button type="button" class="btn">
+                    Get my resume
+                </button>
+            </div>
+        </div>
+        <div class="right">
+            <div class="lang">
+
+            </div>
+            <ul class="language-list">
```
Comment (vickymarz) [inline index.html:109]: - [ ] KIndly add padding around your list of technologies and also reduce the size of the arrows to match the figma design

your design 👇🏽 
image

figma design 👇🏽 
image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @tGemechis7, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
```
Comment (shella12) [inline main.js:63]: - [ ]  As request by the previous reviewer Kindly ☺️ fix the alignment and spacing of the elements of your project cards to match the figma design as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to stick to the figma design. I suggest decreasing the `margin-right` of your teck-stack `ul`   

**_Your_design:_**

new ss

**_Figma_design:_**

ns

File: index.html [hunk 6]
```diff
@@ -1,34 +1,210 @@
...
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="images/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
+    </section>
+    <section id="portfolio" class="project-section">
+
+    </section>
+    <section id="about" class="about">
```
Comment (shella12) [inline index.html:70]: - [ ] As requested by the previous reviewer please make sure the `gap` between each `about-us` section elements looks identical to the the[ figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Dq0auABWvM0gt216-0)

**_Your design_**

Screenshot 2023-02-18 at 7 21 03 AM

**_Figma design_**

Screenshot 2023-02-18 at 8 06 07 AM

File: index.html [hunk 7]
```diff
@@ -1,34 +1,210 @@
...
+        <div class="left">
+            <h1 class="about-me"> About Myself</h1>
```
Comment (shella12) [inline index.html:72]: - [ ] Please make sure the font-size of `about-us` heading looks like [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Dq0auABWvM0gt216-0). I would also suggest putting a `<br>` break element between `about` and `myself` to make it look like figma design

**_Your design_**

Screenshot 2023-02-18 at 7 21 09 AM

**_Figma design_**

Screenshot 2023-02-18 at 8 10 23 AM

File: index.html [hunk 8]
```diff
@@ -1,34 +1,210 @@
...
+            <p class="intro1">
+                Hello I’m a software developer! I can help you build a
+                product , feature or website Look through some of my
+                work and experience! If you like what you see and have
+                a project you need coded, don’t hestiate to contact me.
```
Comment (shella12) [inline index.html:77]: - [ ] Please make sure the `font-size` and `font-weight` of `intro1` paragraph looks like [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Dq0auABWvM0gt216-0). 

**_Your design_**

Screenshot 2023-02-18 at 7 21 13 AM

**_Figma design_**

Screenshot 2023-02-18 at 8 13 13 AM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @tGemechis7 , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- Fixed horizontal scrolling ✅ 

 ## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Please use descriptive commit messages. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). For instance instead of saying `added javascript to save data` say `Add javascript to save data`.Its one of the requirements of the project but please try to keep this in mind for future


Screenshot 2023-02-18 at 8 16 17 AM


- [ ] [OPTIONAL] Great job using the Microverse template for your README and customizing it to reflect your project. 
    - [ ]  [OPTIONAL] Great job adding a link to the MIT Licence into your README, but please remove _`Note:`_ from the LICENCE section of the README. Note is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it


    - [ ] [OPTIONAL] Please remove the default comment like

>             Give credit to everyone who inspired your codebase.  

>            Write a message to encourage readers to support your project  

>             Describe 1 - 3 features you will add to the project.  ,etc from your README. 

They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README.  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -4,31 +4,211 @@
...
+                      <p class="project-description">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <ul class="my-skill">
+                            <li><button title="html">html</button></li>
+                            <li><button title="css">css</button></li>
+                            <li><button title="js">Javascript</button></li>
+                        </ul>
+                        <span class="p-more-information">
+                          <a class="moresee-link btn-action-event">See project</a>
```
Comment (aamirkhan2478) [inline index.html:98]: - [ ] Please make sure your button is exactly the same as the Frigma design and also use all button effects like hover, pressed etc. check below :point_down:  the difference: :+1: 

**Use these button effects:**
Screenshot from 2023-02-19 00-56-06

**You Design:**
Screenshot from 2023-02-20 18-26-28

**Figma Design:**
Screenshot from 2023-02-20 18-26-15

- [ ] Please give some border-radius in your `moresee-link` CSS class in the CSS file.  :slightly_smiling_face:

File: index.html [hunk 10]
```diff
@@ -4,31 +4,211 @@
...
+        <li class="social-icon ">
+            <a><img src="images/m-icon.png" alt="M"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="images/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="images/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
```
Comment (aamirkhan2478) [inline index.html:181]: - [ ] Please use the same Figma design as I mentioned above :point_up: your about section does not look like the Figma design check below :point_down: and kindly fix it :+1: 

**Your Design:**
Screenshot from 2023-02-20 18-28-53

**Figma Design**
Screenshot from 2023-02-20 18-28-30

- [ ] Please use `width: 100%`  to the  `aboutmyself-section` class and also use `border-top-right-radius`  to the `aboutmyself-section` class in your CSS file.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @tGemechis7 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked
✔️ Descriptive PR summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 11]
```diff
@@ -4,31 +4,210 @@
...
+                    </p>
+                    <ul class="my-skill">
+                        <li><button title="html">html</button></li>
+                        <li><button title="css">css</button></li>
+                        <li><button title="js">Javascript</button></li>
+                    </ul>
+                    <span class="p-more-information">
+                      <a class="moresee-link btn-action-event">See project</a>
+                      </span>
+                    </div>
```
Comment (NahnahAJ) [inline index.html:74]: - [ ] Great job closing your tags as the previous reviewer asked. But it looks like after you made those changes, the previous margin/padding you had around your cards was removed and now they look quite different from the Figma template. So kindly add margin/padding to the containers so your work cards each look similar to the Figma design. :100: 
- [ ] Please ensure that you add equal spacing all around the cards so one side doesn't look bigger than the other like you had before :+1: 
- Your Design Now 
image

- The Figma Template
image

- Your Design Before
image

File: index.html (hunk 9 above)
Comment (NahnahAJ) [reply index.html:98]: - [ ] I noticed here that you still didn't make changes to the buttons as the previous reviewer suggested. Kindly include these requirements in your project as they are essential for user interaction. They can help provide clear visual feedback to the user when an action has been successfully completed, which helps to reduce user frustration and increase user satisfaction. :100:

File: index.html [hunk 12]
```diff
@@ -4,31 +4,210 @@
...
+            <li><a class="lang-link">
+            <img class="lang-link-img" src="images/html.png" title="js" alt="javascript icon"> Html</a>
+        </li>
+        </ul>
+        <ul class="job-experience">
+            <li class="job-experience-content">Frameworks</li><li class="job-experience-content">&#62;</li>
+        </ul>
+        <ul class="job-experience">
+            <li class="job-experience-content">Skill</li><li class="job-experience-content">&#62;</li>
+        </ul>
```
Comment (NahnahAJ) [inline index.html:207]: - [ ] It also looks like you made some changes that have changed the initial design you applied to these list items. Kindly re-add them so they look as they did previously :100: 
- Your Design Before
image

- Your Design Now
image

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @tGemechis7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- [ ] No linter errors :heavy_check_mark: 
- [ ] Good use of GitHub flow :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 13]
```diff
@@ -4,31 +4,210 @@
...
+            </li>
+            <li class="social-icon">
+                <a><img src="images/clip-icon.png" alt="fingers"></a>
+            </li>
+        </ul>
+            </span>
+        </section>
+
+         <section class="grid-item grid-item-3 work-section">
+            <div id="work-container">
```
Comment (wayungi) [inline index.html:50]: - [ ] You have created another `div` inside the `<section class="grid-item grid-item-3 work-section">`. The main container of the projects is therefore  `<div id="work-container">` and should have a display:grid

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @tGemechis7 , 
image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tGemechis7** [comment]
i don't undestand your comment can you please explain more

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @tGemechis7 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by tGemechis7** [comment]
thank you ***@***.****

**Pull request merged**